.github/workflows/deployProd.yml

```diff
@@ -17,6 +17,19 @@ env:
 jobs:
```

Comment:
So far I've only modified the deployProd workflow so that it's easier to review, but if the postgres env information needs to be added to the rest of the workflows as well just let me know.